static/locales/en.client.json

```diff
@@ -1,778 +1,735 @@
   },
   "AddNewButton": {
-    "AddNew": "Add New"
-  },
-  "AppHeader": {
-    "HomePage": "Home Page",
-    "Legacy": "Legacy",
-    "PersonalSite": "Personal Site",
-    "TeamSite": "Team Site"
+    "Add New": "Add New"
   },
   "ApiKey": {
-    "AboutToDeleteAPIKey": "You're about to delete an API key. This will cause all future requests using this API key to be rejected. Do you still want to delete?",
-    "AnonymousAPIKey": "This API key can be used to access this account anonymously via the API.",
-    "ByGenerating": "By generating an API key, you will be able to make API calls for your own account.",
-    "ClickToShow": "Click to show",
+    "By generating an API key, you will be able to make API calls for your own account.": "By generating an API key, you will be able to make API calls for your own account.",
+    "Click to show": "Click to show",
     "Create": "Create",
-    "OwnAPIKey": "This API key can be used to access your account via the API. Don’t share your API key with anyone.",
     "Remove": "Remove",
-    "RemoveAPIKey": "Remove API Key"
+    "Remove API Key": "Remove API Key",
+    "You're about to delete an API key. This will cause all future requests using this API key to be rejected. Do you still want to delete?": "You're about to delete an API key. This will cause all future requests using this API key to be rejected. Do you still want to delete?"
   },
   "App": {
     "Description": "Description",
     "Key": "Key",
-    "MemoryError": "Memory Error"
+    "Memory Error": "Memory Error"
+  },
+  "AppHeader": {
+    "Home Page": "Home Page",
+    "Legacy": "Legacy",
+    "Personal Site": "Personal Site",
+    "Team Site": "Team Site"
+  },
+  "AppModel": {
+    "This team site is suspended. Documents can be read, but not modified.": "This team site is suspended. Documents can be read, but not modified."
   },
   "CellContextMenu": {
-    "ResetEntireColumns_one": "Reset entire column",
-    "ResetEntireColumns_other": "Reset {{count}} entire columns",
-    "ResetColumns_one": "Reset column",
-    "ResetColumns_other": "Reset {{count}} columns",
-    "DeleteColumns_one": "Delete column",
-    "DeleteColumns_other": "Delete {{count}} columns",
-    "DeleteRows_one": "Delete row",
-    "DeleteRows_other": "Delete {{count}} rows",
-    "ClearValues": "Clear values",
-    "ClearCell": "Clear cell",
-    "CopyAnchorLink": "Copy anchor link",
-    "FilterByValue": "Filter by this value",
-    "InsertRow": "Insert row",
-    "InsertRowAbove": "Insert row above",
-    "InsertRowBelow": "Insert row below",
-    "DuplicateRows_one": "Duplicate row",
-    "DuplicateRows_other": "Duplicate rows",
-    "InsertColumnRight": "Insert column to the right",
-    "InsertColumnLeft": "Insert column to the left"
+    "Clear cell": "Clear cell",
+    "Clear values": "Clear values",
+    "Copy anchor link": "Copy anchor link",
+    "Delete {{count}} columns_one": "Delete {{count}} columns",
```

Comment:
Something is off with the script, every resource that needs pluralization is wrong. For example here, for one column it should say "Delete column".

Reply:
As I stated in the issue #336, we cannot automatically determine what would be the plural or singular form only by looking at the keys/texte.
You might ask "why bother with plurals and singular" ? Depending on the language, we could have multiples plurals variables, for one, for two maybe for three, some language may have edge case.

Since we cannot determine the value from the key, we decide to put in key the plural form and correct by hands the value for singular or other case.

I'll do the correction by hands on the singular keys which are off